### PR TITLE
v0.2 Rust 1.57 clippy updates

### PR DESCRIPTION
**Pull request by shannynalayna**
Rust 1.57 clippy updates for the 0-2 branch

### CODE REVIEW

**Review by dnewh — APPROVED** [review]

**Review by leebradley — APPROVED** [review]

File: cli/src/actions/products.rs [hunk 1]
```diff
@@ -68,7 +68,11 @@ pub struct GridPropertyValue {
     pub lat_long_value: Option<LatLong>,
 }
 
+/// The `LatLong` value is used by the `GridPropertyValue` `lat_long_value` field and returned to
+/// the user. Rust 1.57 does not recognize the fields of this struct as being read, therefore we
+/// must use the `allow(dead_code)` decorator for this struct.
```
Comment (vaporos) [inline cli/src/actions/products.rs:73]: This comment should probably be "//" not "///" as it isn't intended to be API doc.

File: sdk/src/workflow/state.rs [hunk 2]
```diff
@@ -15,6 +15,9 @@
 #[derive(Clone)]
 pub struct WorkflowState {
     name: String,
+    #[allow(dead_code)]
```
Comment (vaporos) [inline sdk/src/workflow/state.rs:18]: This could probably be removed instead, since its not used in v0.2.

Comment (shannynalayna) [reply sdk/src/workflow/state.rs]: Removed this field in the next commit

File: cli/src/actions/products.rs (hunk 1 above)
Comment (shannynalayna) [reply cli/src/actions/products.rs]: Updated this comment to use "//" instead

**Review by dnewh — APPROVED** [review]

**Review by agunde406 — APPROVED** [review]

**Pull request merged**